microraiden/microraiden/proxy/paywalled_proxy.py

```diff
@@ -148,3 +132,14 @@ def gevent_error_handler(context, exc_info):
         if isinstance(e, ssl.SSLError) and e.reason == 'HTTP_REQUEST':
             return
         gevent.get_hub().handle_system_error(exc_info[0], exc_info[1])
+
+    def add_paywalled_resource(self, cls: Expensive, url: str, price: int, *args, **kwargs):
```

Comment:
`cls` would now be `Type[Expensive]` because it takes a class. Wouldn't have noticed but my IDE complained. Type hints in flask appear to be wrong. It will still complain about the `self.api.add_resource()` call though but I guess that's just because flask uses older type hints.

Also, `price` would technically be `Union[int, Callable[[str], int]]`

Sorry for being a type hint pedant.